github-action/pkg/qovery_api_client.go

```diff
@@ -54,12 +142,30 @@ func NewUnknownEnvironmentStatus(id string) EnvironmentStatus {
 	}
 }
 
-func NewEnvironmentStatus(id string, state EnvStatus, message string, serviceDeploymentStatus string) EnvironmentStatus {
-	return EnvironmentStatus{
+func NewUnknownApplicationStatus(id string) ApplicationStatus {
+	return ApplicationStatus{
+		ID:                      id,
+		State:                   AppStatusUnknown,
+		Message:                 "",
+		ServiceDeploymentStatus: "",
+	}
+}
+
+func NewUnknownContainerStatus(id string) ContainerStatus {
+	return ContainerStatus{
+		ID:                      id,
+		State:                   ContStatusUnknown,
+		Message:                 "",
+		ServiceDeploymentStatus: "",
+	}
+}
+
+func NewUnknownDatabaseStatus(id string) DatabaseStatus {
+	return DatabaseStatus{
 		ID:                      id,
-		State:                   state,
-		Message:                 message,
-		ServiceDeploymentStatus: serviceDeploymentStatus,
+		State:                   DbStatusUnknown,
+		Message:                 "",
```

Comment:
I did it the same way it was done to others. I didn't want to change the way things were done. I think it will have to be completely be rewritten with the come of deployments groups and the need of interfaces